app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM alpine
```

Comment:
We need to stick on a specific Alpine version. Let's go for `alpine:3.6`, which seems to be the latest.